### DISCUSSION TITLE
Windows crashed - reinstalled without losing data but Sandboxie lost link to its sandboxes

### DISCUSSION THREAD

**Discussion by bwayan** [sandboxie-plus/Sandboxie#3649] [General]
Hello guys

As stated in the title, my windows crashed and I had to reinstall it, while keeping the option to keep my personal data.
All programs were somehow removed from registries ect etc ...

Now I am launching Sandboxie; but while my sandboxes with their data are still on my drive, they do not appear anymore in my main program.
How to link them back to Sandboxie ?

I searched on the forum for that point, but impossible to find anything.

Cheers

**Comment by offhub** [comment]
> [!WARNING]
> Don't forget to backup the `C:\Sandbox` folder, just in case.

You can use the sandbox data on disk by creating sandboxes with the same name. However, if you have not backed up the `C:\Windows\Sandboxie.ini` file, you will have to re-configure the settings you made from the interface.
